### ISSUE TITLE
Must commandPoolResetCommandBuffer be supported?

### ISSUE THREAD

**Issue by etomzak** [KhronosGroup/VulkanSC-Docs#4]
In Appendix H, under Modifications, in the Command Buffers entry, the spec states

```quote
* `commandPoolResetCommandBuffer` must be supported [SCID-8].
```

A few lines below, it says

```quote
If `commandPoolResetCommandBuffer` is not supported, `commandBuffer` must be in the initial state [SCID-8].
```

Unless I'm missing something, these contradict, and one of the lines should be removed.

**Comment by dgkoch** [comment]
Yes - that was misleading, and we already have a fix that'll be in the next spec update. 
It was cause by the first line not having enough context. 

It has been changed to: 
```
-  * <<limits-commandPoolResetCommandBuffer,pname:commandPoolResetCommandBuffer>>
-    must: be supported <<SCID-8>>.
+  * If <<limits-commandPoolResetCommandBuffer,pname:commandPoolResetCommandBuffer>>
+    is not supported <<SCID-8>>, flink:vkResetCommandBuffer must: not be called.

```
And the second line has been changed to: 
```
   * If
     <<limits-commandPoolResetCommandBuffer,pname:commandPoolResetCommandBuffer>>
     is not supported, pname:commandBuffer must: be in the
-    <<commandbuffers-lifecycle, initial state>> <<SCID-8>>.
+    <<commandbuffers-lifecycle, initial state>> when flink:vkBeginCommandBuffer
+    is called <<SCID-8>>.

```

**Comment by dgkoch** [comment]
This has been addressed in the 1.0.12 spec update.
